internal/pkg/cli/svc_init.go

```diff
@@ -388,13 +388,13 @@ func (o *initSvcOpts) askSvcPort() error {
 	return nil
 }
 
-func (o *initSvcOpts) getRelativePath() (string, error) {
-	copilotDirPath, err := o.ws.CopilotDirPath()
+func getRelativePath(ws copilotDirGetter, dockerfilePath string) (string, error) {
```

Comment:
Can we rename this one to be `relativeDockerfilePath` to make this function name more descriptive? Also could you move this function to `cli.go` since this is not a function specifically for `svc init`?

Reply:
Yup, done. Thanks!